### DISCUSSION TITLE
file.ts生成的更新日志并没有生成新加入的应用

### DISCUSSION THREAD

**Discussion by adproqwq** [aoguai/subscription#52] [General]
例如：原订阅中没有应用A的规则，现在加入了应用A的规则，但是更新日志中并没有应用A的变更记录。
可否能优化下代码，或者把如何修改的思路分享一下？

**Comment by aoguai** [comment]
https://github.com/aoguai/subscription/blob/593f4c35f39c31d8bbfc54dd8594feaeb242233b/src/file.ts#L545-L577

593f4c35f39c31d8bbfc54dd8594feaeb242233b

**Comment by adproqwq** [comment]
感谢
